airbyte-integrations/bases/standard-source-test/build.gradle

```diff
@@ -13,6 +13,7 @@ import org.jsoup.Jsoup;
 
 dependencies {
     implementation project(':airbyte-db:db-lib')
+    implementation project(':airbyte-api')
```

Comment:
Could this be a `testImplementation` rather than `implementation`? Is it only a dependency because it is used in the test?

Reply:
Yes exactly... removed